repy.py

```diff
@@ -331,6 +331,7 @@ def main():
   ### anywhere if this is repurposed...
   usage = "USAGE: repy.py [options] resource_file program_to_run.r2py [program args]"
   parser = optparse.OptionParser(usage=usage)
+  parser.disable_interspersed_args()
```

Comment:
You may even add a comment explaining the importance of restoring traditional Unix syntax and what can go wrong if command-line options are consumed by repy.py.  Not adding a comment here seems like an easy way for a future developer to miss the importance of this behaviour.

Example comment:
`disable_interspersed_args()` restores traditional Unix syntax, where option parsing stops with the first non-option argument.  `disable_interspersed_args()` should be used if you have a command processor  which runs another command which has options of its own and you want to make sure these options don’t get confused. For example, each command might have a different set of options.